docker/setup.sh

```diff
@@ -15,12 +15,14 @@
 
 conda create -y -n beakerx 'python>=3' nodejs pandas openjdk maven
 source activate beakerx
-conda install -y -c conda-forge ipywidgets
+conda install -y -c conda-forge ipywidgets jupyterhub jupyterlab
 
 rm -r /home/beakerx/beakerx/js/node_modules
 rm -r /home/beakerx/beakerx/js/dist
 
 (cd beakerx; pip install -e . --verbose)
+jupyter labextension install @jupyter-widgets/jupyterlab-manager
+jupyter labextension install beakerx-jupyterlab
```

Comment:
this isn't right, it would install from npm.
we should install the code we just built from the local repository.
see the README for now to do that, basically need a path to js/lab directory.